tests/components/sonos/test_sensor.py

```diff
@@ -45,7 +45,7 @@ async def test_battery_attributes(hass, async_autosetup_sonos, soco):
     )
 
 
-async def test_battery_on_S1(hass, async_setup_sonos, soco, battery_event):
+async def test_battery_on_S1(hass, async_setup_sonos, soco, device_properties_event):
```

Comment:
```suggestion
async def test_battery_on_s1(hass, async_setup_sonos, soco, device_properties_event):
```

Reply:
The "S1" refers to the firmware revision which is normally stylized with a capital.

Reply:
Usually we only use lowercase in the function name, but can stylize in the docstring.